tests/suite/editMercurialVaultDepositorySuite.ts

```diff
@@ -7,56 +7,73 @@ import {
 } from '@uxd-protocol/uxd-client';
 import { getConnection, TXN_OPTS } from '../connection';
 import { editMercurialVaultDepositoryTest } from '../cases/editMercurialVaultDepositoryTest';
-
-export const editMercurialVaultDepositorySuite = async function (
-  controllerAuthority: Signer,
-  user: Signer,
-  payer: Signer,
-  controller: Controller,
-  depository: MercurialVaultDepository
-) {
+import {
+  MERCURIAL_USDC_DEVNET,
+  MERCURIAL_USDC_DEVNET_DECIMALS,
+  uxdProgramId,
+} from '../constants';
+
+export const editMercurialVaultDepositorySuite = async function ({
+  authority,
+  controller,
+}: {
+  authority: Signer;
+  controller: Controller;
+}) {
+  let depository: MercurialVaultDepository;
   let beforeDepository: MercurialVaultDepositoryAccount;
 
   describe('Edit mint/redeem', () => {
-    // Snapshot the initial depository values
     before(async () => {
+      depository = await MercurialVaultDepository.initialize({
```

Comment:
qq: mentionned on this thread:
https://github.com/UXDProtocol/uxd-program/pull/225/files#r1087621234

this could be inside of a utility function right?

Reply:
yes!